.github/workflows/msrv.yml

```diff
@@ -19,10 +19,9 @@ jobs:
       - uses: actions-rs/toolchain@v1
         with:
           profile: minimal
-          toolchain: stable
-      - name: install cargo hack
-        run: cargo install cargo-hack
+          default: true
+          toolchain: 1.49
       - name: cargo test msrv..
         run: |
           cd compact_str
-          cargo hack test --feature-powerset --optional-deps --version-range 1.49..
```

Comment:
Why did you get rid of `cargo hack`? It's quite useful in making sure `compact_str` works for all versions of rustc since the MSRV, and all different combination of features?

Reply:
They are taking too long to run, probably because of the new features I added

Reply:
OK, I've reverted the changes to this workflow.